code/script.js

```diff
@@ -1,51 +1,117 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat');
+const chat = document.getElementById('chat')
+const inputWrapper = document.getElementById('input-wrapper')
+const input = document.getElementById('input')
+const sendBtn = document.getElementById('send')
 
-// If you need any global variables that you can use across different functions, declare them here:
+let botQuestion = 1
 
+const botReply = (msg) => {
+  showMessage(msg, 'bot')
+}
 
-// Declare your functions after this comment
+const userReply = (msg) => {
+  showMessage(msg, 'user')
+}
 
-// This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
-  // the if statement checks if the sender is 'user' and if that's the case it inserts an html senction inside the chat with the posted message
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/cloud-crowd-bot.svg" alt="User" />  
       </section>
     `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
   } else if (sender === 'bot') {
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/cloud-crowd-bot.svg" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight;
+  chat.scrollTop = chat.scrollHeight
+}
+
+const nextQuestion = (message) => {
+  console.log('questionNumber', botQuestion)
+
+  if (botQuestion === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => showAltitudeTypes(message), 1000)
+  } else if (botQuestion === 2) {
+    userReply(message)
+    setTimeout(() => showMenu(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(cloudType, 1000)
```

Comment:
Here it looks like the setTimeout for the cloudType is broken, should it look like the ones above? 
setTimeout(() => cloudType(message), 1000) ? I'm not sure though...